libs/core/src/components/pds-sortable-item/pds-sortable-item.tsx

```diff
@@ -0,0 +1,46 @@
+import { Component, Host, h, Prop } from '@stencil/core';
+
+/**
+ * @slot sortable-item-actions - Content is placed within the `pds-sortable-item__actions` element as children. This slot is only rendered if `actions` is set to `true`.
+ */
+@Component({
+  tag: 'pds-sortable-item',
+  styleUrl: 'pds-sortable-item.scss',
+  scoped: true,
+})
+export class PdsSortableItem {
+  /**
+   * Determines whether `sortable-item-actions` slot should be enabled.
+   * @defaultValue false
+   */
+  @Prop() actions = false;
```

Comment:
I don't hate it, but I would say we should add it to the housekeeping doc, and if we all agree we can revisit and update all components.

Reply:
Okay after more thinking on this one, I do think I should rename it, but to something like `enableActions` since it's not an actual check if actions exist, it just enables the slot.

Reply:
I agree that we should have a conversation on this. I would love if we can continue to have consistent naming conventions.